Java examples for Amazon Elastic Transcoder

lmatuszek opened the pull request:
Adding Java examples for using Amazon Elastic Transcoder.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

jschwarzwalder commented:
For source author we can list your username or simply put AWS.

lmatuszek commented:
> 
> 
> For source author we can list your username or simply put AWS.

The samples were written by an AWS dev a few years ago. Let's use AWS.

soo-aws commented on java/example_code/elastictranscoder/src/main/java/aws/example/elastictranscoder/RetryRequest.java:
```diff
@@ -0,0 +1,41 @@
+ * permissions and limitations under the License.
+ */
+currentRetry = 0
+DO
```
i don't think this is a snippet for java. is it supposed to be part of a different language PR?

soo-aws commented:
Could you add the following snippet tags with appropriate snippet names? 
//snippet-start:[]
//snippet-end:[]
The format we've been using is 
service.language.descriptive_name.snippet_type 

for example: transcoder.java.delete_error.complete

Also I think the RetryRequest.java snippet is not java code. Did you mean to submit for a different language?

lmatuszek commented:
No, double checking the docs the Retry was a workflow example and not an actual code sample. I've removed it, and added sourceauthor and snippet start/ends.
